src/steps/2.addModules/moduleConfigs.ts

```diff
@@ -386,12 +411,15 @@ export const moduleConfigs: Record<Modules, ModuleConfig> = {
       "- [ ] Auth: Configure your auth providers to the [NuxtAuthHandler](./server/api/auth/[...].ts)",
       "- [ ] Auth, optional: Enable global protection by setting `enableGlobalAppMiddleware: true` in [your nuxt.config.ts](./nuxt.config.ts). Delete the local middleware in the [protected.vue](./pages/protected.vue) page if you do"
     ],
-    htmlForIndexVue: `<p>
-      Checkout the page protected by \`nuxt-auth\` here:
-      <nuxt-link to="/protected" class="underline text-blue">
-        Click me to test the auth setup!
-      </nuxt-link>
-    </p>`
+    indexVue: {
+      html: generateModuleHTMLSnippet(
+        "nuxt-auth",
+        "Nuxt user authentication and sessions via NextAuth.js. nuxt-auth wraps NextAuth.js to offer the reliability & convenience of a 12k star library to the nuxt 3 ecosystem with a native developer experience (DX)",
+        "auth__card",
+        "https://sidebase.io/nuxt-auth/getting-started"
+      ),
+      css: ".auth__card { background: radial-gradient(#0FCF97, #0B9A71); }",
```

Comment:
let's also generate the CSS in `generateModuleHTMLSnippet` and have `generateModuleHTMLSnippet` return an object like:
```
{
  html: ...,
  css: ...
}
```


then here you can do:
```
indexVue: {
  ...generateModuleHTMLSnippet(...)
}
```


-> this makes way more sense as css and html are tightly coupled by nature (e.g.: classname MUST match between html and css), so doing it this way will avoid random errors down the road